<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RequestCard</class>
 <widget class="QWidget" name="RequestCard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>50</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>340</width>
    <height>40</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>50</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QLabel" name="usernameLabel">
     <property name="font">
      <font>
       <pointsize>17</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(60, 45, 50);</string>
     </property>
     <property name="text">
      <string>Никнейм</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="addButton">
     <property name="minimumSize">
      <size>
       <width>45</width>
       <height>45</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>45</width>
       <height>45</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(60, 45, 50); 
border: 5px solid rgb(60, 45, 50);
border-radius: 10;</string>
     </property>
     <property name="text">
      <string>+</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="removeButton">
     <property name="minimumSize">
      <size>
       <width>45</width>
       <height>45</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>45</width>
       <height>45</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(60, 45, 50); 
border: 5px solid rgb(60, 45, 50);
border-radius: 10;</string>
     </property>
     <property name="text">
      <string>x</string>
     </property>
     <property name="iconSize">
      <size>
       <width>35</width>
       <height>20</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
